app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Remove this file, as:
- This file does not belong here.  Manifest should be in app/manifests directory.
- This file is incomplete.